contacts_vs_distance.ipynb

```diff
@@ -44,7 +44,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    cvds = []
</pre><p>cvds_smoothed?</p><p> </p><p>also can add one comment here</p><p><code># expected calculation for the smoothed data</code></p><p> </p><p>also see comment above about need to modify expected_cis so that it can correctly output counts.avg.smoothed (and not just smoothed the balance columns) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>